content/tutorial/tutorial.md

```diff
@@ -581,19 +588,19 @@ function Square(props) {
 }
 ```
 
-We have changed `this.props` to `props` both times it appears.
+हमने `this.props` को `props` में बदल दिया है, दोनों जगह जहाँ इसका इस्तेमाल हो रहा था।
```

Comment:
this could be condensed into a single sentence?

Reply:
`हमनें दोनों जगह `this.props` को `props` में बदल दिया है जहाँ इसका इस्तेमाल हो रहा था`

Reply:
हमनें दोनों जगह `this.props` को `props` में बदल दिया है is enough?